readyplayerme/Unity-Avatar-Loader
# [3407583725] Auto include shader variants + material properties fix

## [3407583725](https://ready-player-me.monday.com/boards/2563815861/pulses/3407583725)

## Description

-   setup shader variants collection to better solve the missing shaders problem as per gltFast docs https://github.com/atteneder/glTFast/blob/main/Documentation~/ProjectSetup.md#materials-and-shader-variants
- also fixed an issue with material property names




## Changes

#### Added

-  shader variants collection

#### Updated

-   material property names



## How to Test

- add via package manager using this URL 
https://github.com/readyplayerme/Unity-Core.git#feature/shader-variants
- check for errors and check that shader variants are added into graphics settings (and avatar loading works in builds)



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
